public-user/molecule/default/tests/test_default.py

```diff
@@ -1,14 +1,16 @@
-import testinfra.utils.ansible_runner
 import json
+import os
+import testinfra.utils.ansible_runner
 
 testinfra_hosts = testinfra.utils.ansible_runner.AnsibleRunner(
-    '.molecule/ansible_inventory').get_hosts('all')
+    os.environ['MOLECULE_INVENTORY_FILE']).get_hosts('all')
+
 
-OMERO = '/opt/omero/web/OMERO.web/bin/omero'
+OMERO = '/opt/omero/server/OMERO.server/bin/omero'
```

Comment:
Since you're touching this line you might as well delete it, doesn't look like it's used anywhere.